### DISCUSSION TITLE
Behavior Packs and Bedrock Resource Packs

### DISCUSSION THREAD

**Discussion by noawx** [modrinth/.github#141] [Feature requests]
Adding two categories for Resource Packs (Java & Bedrock) and adding a category for Behavior Packs could increase the amount of users since both Java and Bedrock players could download files from Modrinth.

(unless it is already planned)

**Comment by blank3times** [comment]
Bedrock is already on the roadmap, and I would like to suggest an alternative to having two categories for resource packs. I think version support for Bedrock texture packs under the resource packs category would be a better choice, and a good way to add an amount of support in a shorter timespan.

This would include:
-  Allowing `.mcpack` file extensions for version uploads
- Allowing `manifest.json` as an alternative to `pack.mcmeta`
- Adding a "Bedrock Edition" label option to the versions metadata (akin to Planet Minecraft, since Bedrock doesn't have old versions on its launcher), and/or all the Bedrock updates (if you want to go overboard; this is tedious.)

I am suggesting this as an alternative to having two separate categories because this would allow for a pack to have just one page with both editions---sharing statistics and having the separate downloads in the versions. I would not consider it another game like the roadmap implies (many packs are made for both editions at once). And while Bedrock is "slated for the future", I think this solution can provide an effective form of support.

**Comment by Felix14-v2** [comment]
I think combining Minecaraft and Bedrock Edition isn't a good idea because of the extremely difference they have. Instead it would be more logical to have Bedrock Edition as a separate game (yeah, Modrinth pack format can support different games in .mrpack). Anyway, sooner or later the Modrinth team wanted to start adding support for other games, and Bedrock Edition is a great reason for this.

**Comment by triphora** [comment]
Support for other games is already on [our roadmap](https://roadmap.modrinth.com), please do not make discussions for roadmap items. If you are a Bedrock developer, please DM me on Discord (`@triphora`) as we have a private channel on our Discord for planning how we want Bedrock to be done exactly.
